filters/tee/tee.go

```diff
@@ -0,0 +1,172 @@
+package tee
```

Comment:
the package should have a documentation, at least a headline. See: https://godoc.org/github.com/zalando/skipper#pkg-subdirectories